apple/Handlers/RNNativeViewHandler.mm

```diff
@@ -216,7 +216,7 @@ @implementation RNNativeViewGestureHandler
 
 - (instancetype)initWithTag:(NSNumber *)tag
 {
-  RCTLogWarn(@"Manual gesture handler is not supported on macOS");
+  RCTLogWarn(@"Native view gesture handler is not supported on macOS");
```

Comment:
I think this looks better. Let me know what you think.

```suggestion
  RCTLogWarn(@"NativeViewGestureHandler is not supported on macOS");
```

Reply:
Yup, imo it looks better for long names. When I wrote code for warnings for the first time I followed the message from hover handler on tvOS, but now I see that we do different for force touch (one single word).

Reply:
Yeah, it looks nicer. I'm fine with both, merging this one and changing them all in a follow-up, or just changing them in this PR.